astroid/brain/brain_builtin_inference.py

```diff
@@ -257,10 +270,12 @@ def _container_generic_transform(
     iterables: tuple[type[nodes.BaseContainer] | type[ContainerObjects], ...],
     build_elts: BuiltContainers,
 ) -> nodes.BaseContainer | None:
+    elts: Iterable
```

Comment:
This contradicts line 301.

Reply:
Line 301 is the `klass` argument? Having trouble following.

Reply:
Oops, sorry! New code line 300

Reply:
Got it now, thanks!